frontend/src/app/features/in-app-notifications/entry/reminder-alert/in-app-notification-reminder-alert.component.html

```diff
@@ -0,0 +1,8 @@
+<op-in-app-notification-relative-time
+  [notification]="reminderAlert"
+  [hasActorByLine]="false"
+/>
+<span
```

Comment:
<img width="389" alt="Bildschirmfoto 2024-12-11 um 13 25 51" src="https://github.com/user-attachments/assets/bc0d878f-2f32-4479-ae4d-a594e7135c7e">

I still see the issue raised by wieland in our last meeting as well - a longer note text gets hidden on mobile